modules/ROOT/pages/tips-and-tricks.adoc

```diff
@@ -160,3 +160,16 @@ or setup shell `alias`es as needed to make them available to the CLI like so:
   $ alias evince="flatpak run org.gnome.Evince"
     # or alias evince="org.gnome.Evince"
   $ evince
+
+== Enabling TPM2 for LUKS 
+
+It is possible to use the TPM2 device in your machine to automatically decrypt your LUKS-encrypted device. 
+
+First, enable the needed initramfs and kernel features: 
+
+  $ sudo rpm-ostree kargs --append=rd.luks.options=tpm2-device=auto
+  $ sudo rpm-ostree initramfs --enable --arg=-a --arg=systemd-pcrphase
+
+Then, enroll the device: 
+
+  $ sudo systemd-cryptenroll --tpm2-device=auto --tpm2-pcrs=0+7 /your/boot/device
```

Comment:
I think `/your/boot/device` is confusing. It makes it sound like you should specify your `/boot` or ESP device here, when actually it's your LUKS block device.

Maybe `/your/luks/device` is better?

And then also add a follow-up paragraph that explains you can determine which device that is by running `sudo cryptsetup status /dev/mapper/luks-*` or looking in `/etc/crypttab`.

Reply:
added some steps around finding the proper device, changed my shorthand for the device location